vars/perfRegressionParallelPipeline.groovy

```diff
@@ -70,6 +70,9 @@ def call(Map pipelineParams) {
             string(defaultValue: "${pipelineParams.get('k8s_scylla_operator_docker_image', '')}",
                    description: 'Scylla Operator docker image',
                    name: 'k8s_scylla_operator_docker_image')
+            string(defaultValue: "${pipelineParams.get('k8s_deploy_monitoring', '')}",
```

Comment:
I believe it comes from commit that is already in master

Reply:
> @KnifeyMoloko In addition to the comments above, I would like to see some results here.

@roydahan 
I've collected a few results here: https://docs.google.com/spreadsheets/d/1q1-V1BrZhJ9abVowaktBKPh5IFVrTnXsdP2OReOQtyY/edit?usp=sharing

This was mostly for my own tweaking the workload params with na i3. I'll add the r5 results there when I have them.

Reply:
I don't see it in master at the moment, and IIRC this was failing my Jenkins runs.